db/db.js

```diff
@@ -156,6 +156,33 @@ async function getSessionWithSessionId(id, clientParam) {
   }
 }
 
+async function getSessionWithSessionIdAndAlertApiKey(sessionId, alertApiKey, clientParam) {
+  try {
+    const results = await helpers.runQuery(
```

Comment:
Slowly starting to use the version of this method that is in `brave-alert-lib`. It has the same implementation but requires the `pool` object as a parameter.